fb_instant_articles.module

```diff
@@ -77,9 +78,31 @@ function fb_instant_articles_help($path, $arg) {
       $output .= '  <li>' . t('Check out the <a href="@blog_url" target="_blank">Instant Articles blog</a> and <a href="@sign_up_url" target="_blank">sign up</a> to receive notifications of important updates.', array('@blog_url' => 'https://developers.facebook.com/ia/blog/', '@sign_up_url' => '')) . '</li>';
       $output .= '  <li>' . t('To give other members of your team access to the <a href="@tools_url" target="_blank">Instant Articles tools</a>, assign them <a href="@roles_url" target="_blank">page roles here</a>.', array('@tools_url' => '', '@roles_url' => '')) . '</li>';
       $output .= '</ol>';
+      return $output;
       break;
   }
-  return $output;
+}
+
+/**
+ * Adds markdown module support for displaying README files as Drupal help text.
+ *
+ * @param string $string
+ *   The unprocessed README file contents string.
+ *
+ * @return string|false
+ *   A string containing the markdown filtered text, or FALSE.
+ *
+ * @see fb_instant_articles_help()
+ */
+function fb_instant_articles_help_markdown($string) {
+  if (module_exists('markdown')) {
```

Comment:
@m4olivei heh yeah, that was suggested by @vgardner. I was thinking we may want to add the admin help text into a md file of it's own (from [here](https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/33/files#diff-062ec483c58e7768ec4bac083c4b499fR61)), but we can follow up on that if we want to.